README.md

```diff
@@ -20,14 +20,26 @@ We assume that you have Galaxy running and docker installed in your server/lapto
 
 * Configure the `job_conf.yml` under `lib/galaxy/config` to enable the docker for the environment you want the Ludwig related job running in. This is an example:
 ```
+runners:
```

Comment:
it is good to have this. but can you change "This is an example" to something like "This is an example of job_conf.yml for a local galaxy."? Thank you!

Reply:
Changes made